network/network.go

```diff
@@ -0,0 +1,10 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// TODO(kortschak): Implement:
+// * edge-weighted PageRank and HITS
+// * other centrality measures
```

Comment:
That's the right problem to solve. I don't know if that specific paper is the best algorithm.